Stanford/JumpstartEngineering/Install/Courses/DisableCourses.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @file
+ * Enable Courses
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Courses;
+
+/**
+ *
+ */
+class DisableCourses extends \ITasks\AbstractInstallTask {
+
+  /**
+   * Disable Courses module on JSE sites
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    module_load_include('module', 'redirect', 'redirect.module');
+    $modules = array( 'stanford_course_views', 'stanford_courses_administration',
```

Comment:
Looking at the api function it appears you can.
https://api.drupal.org/api/drupal/includes%21module.inc/function/module_disable/7.x

I don't see anything in there that is preventing a module from being disabled if it has enabled dependencies.